adafruit_veml7700.py

```diff
@@ -196,17 +202,17 @@ def __init__(self, i2c_bus, address=0x10):
         else:
             raise RuntimeError("Unable to enable VEML7700 device")
 
-    def integration_time_value(self):
+    def integration_time_value(self) -> int:
         """Integration time value in integer form. Used for calculating :meth:`resolution`."""
         integration_time = self.light_integration_time
         return self.integration_time_values[integration_time]
 
-    def gain_value(self):
+    def gain_value(self) -> int:
```

Comment:
This should return `float` based on the dictionary.

Reply:
oops, I see that now.